src/BloomBrowserUI/bookEdit/toolbox/readers/readerToolsModel.ts

```diff
@@ -1191,7 +1191,7 @@ export class ReaderToolsModel {
           axios.get<string>('/bloom/api/readers/io/allowedWordsList', { params: { 'fileName': stage.allowedWordsFile } })
               .then(result => this.setAllowedWordsListList(result.data, index));
       }
-    });
+    }.bind(this));  // this bind is needed on Linux if not on Windows
```

Comment:
This would help me in the future:

// During Linux testing, the this (_this in the TS converted to JS), in the axios.get callback was undefined without this bind(). (see BL-3498)

thx